schema-library-events/src/main/avro/alertConfig.avdl

```diff
@@ -0,0 +1,31 @@
+@namespace("com.homeaway.digitalplatform.streamregistry")
+
+protocol AlertProtocol {
+
+  /**
+  * S3 Connector gets data from Kafka and dumps it in S3
```

Comment:
This isn't specific to S3

Reply:
I agree. My bad, it was a copy-over error.